client/public/styles/navbar.scss

```diff
@@ -1,11 +1,25 @@
 .header-row {
-  padding: 1vh;
   height: 10vh;
+  background-image: radial-gradient(
+      50% 141%,
+      rgba(11, 47, 129, 0.6) 1%,
+      rgba(43, 134, 161, 0.651) 100%
+    ),
+    url("/asset/images/splash.jpg");
+  vertical-align: top;
+  padding: 1vh;
+
+  .col-1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
 }
 
 .header-img {
-  width: 70%;
-  height: auto;
+  // width: 70%;
+  height: 8vh;
```

Comment:
Not sure why we change here to using vh and vw instead of consistently using em. If there's a better case for this, let me know.